litecommands-fabric/build.gradle.kts

```diff
@@ -16,8 +16,8 @@ repositories {
 
 dependencies {
     // LiteCommands
+    api(project(":litecommands-framework"))
```

Comment:
This still works and is a better solution for development (IntelliJ will support refactoring etc.)

Reply:
In `a4c9207` I mistake message of the commit.